Sources/Sentry/UIViewControllerHelper.m

```diff
@@ -0,0 +1,34 @@
+#import "UIViewControllerHelper.h"
+
+@implementation UIViewControllerHelper
```

Comment:
`m`: We have duplicate code in `SentryBreadcrumbTracker`. Can we please fix that?

Reply:
Definitely. I create this class because SentryBreadcrumbTracker already used this method, and I did not want duplication. Looks like I forget to replace in the original reference.